engine/src/main/java/io/camunda/zeebe/engine/state/ZbColumnFamilies.java

```diff
@@ -66,7 +66,10 @@
 
   // process message subscription
   PROCESS_SUBSCRIPTION_BY_KEY,
+  // migration start
+  @Deprecated // only used for migration logic
```

Comment:
(optional) We could add the comment directly in the annotation. So, it is displayed directly in the IDE.

```suggestion
  @Deprecated("should not be used anymore except for migration logic")
```

Reply:
Seems we are thinking of different annotations. The Java 11 one has `since` and `forRemoval` as fields

Reply:
You're right :sweat_smile: